README.md

```diff
@@ -2,7 +2,7 @@
 
 > Informações gerais sobre o Training Center
 
-Esta comunidade **não possui meios de conseguir renda**, porém, você pode doar mensalmente para nosso grupo através do [apoia.se](http://apoia.se/training-center) e garantir a continuação do funcionamento das nossas ferramentas. ;P
```

Comment:
Pode explicar o objetivo desta edição?
Posso estar errado, mas acredito que `porém` vem sempre entre vírgulas. Não vejo porque remover o emote, da uma descontraída que contrasta com o tema sério.

Reply:
Você está certo, porém sem virgula é somente no início da frase

Reply:
Boa, melhor!

@nicholasess você pode me explicar o motivo da remoção do emote?

Reply:
O emoji não traz confiabilidade para o texto e é exatamente isso que o "Sobre" quer passar. Deixa a informalidade para comunicação interna, aquele jogo de cintura para lidar com as pessoas no dia a dia e no texto, seja interessante, formal e legível.

Reply:
Será que o readme precisa dessa formalidade, em uma comunidade informal como o TC ?
E isso foge um pouco do contexto do PR, que seria para correções do readme, concorda ?

Reply:
Eu acho que não @nicholasess, o emoji é justamente para que possamos criar um ambiente que seja ao mesmo tempo descontraído e sério, para que as pessoas saibam que elas podem se divertir no CT mas tem de respeitar o nosso código de conduta também.

Reply:
Acho que esta discussão poderia estar em outra PR

Reply:
Galera, não vamos deixar as coisas super formais, já é difícil as pessoas lerem, se tornarmos isso ainda mais pesado, ai que ninguem vai ler. Fora que a ideia é ter um vídeo explicativo sobre o TC. Logo isso vai ficar muito mais acessível.